dist/avg_dom_depth.js

```diff
@@ -12,7 +12,8 @@ function avgDomDepth() {
 function numParents(elem) {
   var n = 0;
   if ( elem.parentNode ) {
-    while ( elem = elem.parentNode) {
+    /* eslint-disable-next-line no-cond-assign */
+    while (elem = elem.parentNode) {
```

Comment:
```suggestion
    while ((elem = elem.parentNode)) {
```

Reply:
Can you explain why this would be needed out of interest?

Reply:
Just an alternative approach that I found easier than adding ignore.
It hides assignment within parenthesis and linter is fine.

Reply:
Oh I missed that you'd removed the linter override too. Nice work around (if a little bit of a cheat!).